framework/doc/content/syntax/Controls/index.md

```diff
@@ -83,13 +83,103 @@ the syntax for a parameter name is specified as: `block/object/name`.
 As shown in [controls_example] an asterisk ("*") can be substituted for any one of these three
 "names", doing so allows multiple parameters to match and be controlled simultaneously.
 
-In similar fashion, object names can be defined (e.g., as in the
-[`TimePeriod`](/TimePeriod.md)) object. In this case, the general name scheme is the same
+In similar fashion, object names can be requested by controls (e.g., as in the
+[`TimePeriod`](/TimePeriod.md)). In this case, the general name scheme is the same
 as above but the parameter name is not included.
 
 In both cases there is an alternative form for defining an object and parameter names:
 `base::object/name`. In this case "base" is the MOOSE base system that the object is derived from.
-For example, `Kernel::diff/coef`.
+For example, `Kernel::diff/coef`. All MOOSE "bases" are listed bellow:
+
+- ArrayAuxKernel,
+- ArrayKernel,
+- AuxKernel,
+- AuxScalarKernel,
+- BoundaryCondition,
+- Constraint,
+- Damper,
+- DGKernel,
+- DiracKernel,
+- Distribution,
+- EigenKernel,
+- Executioner,
+- Executor,
+- Function,
+- FVBoundaryCondition,
+- FVInterfaceKernel,
+- FVKernel,
+- Indicator,
+- InitialCondition,
+- InterfaceKernel,
+- Kernel,
+- LineSearch,
+- Marker,
+- MaterialBase,
+- MeshGenerator,
+- MooseMesh,
+- MoosePartitioner,
+- MoosePreconditioner,
+- MooseVariableBase,
+- MultiApp,
+- NodalKernel,
+- Output,
+- Postprocessor,
+- Predictor,
+- Problem,
+- RelationshipManager.,
+- Reporter,
+- Sampler,
+- ScalarInitialCondition,
+- ScalarKernel,
+- Split,
+- TimeIntegrator,
+- TimeStepper,
+- Transfer,
+- UserObject,
+- VectorAuxKernel,
+- VectorInterfaceKernel,
+- VectorKernel,
+- VectorPostprocessor,
+
+MOOSE allows objects to define a `tag` name to access its controllable parameters with their `control_tags` parameter.
+
+!listing test/tests/controls/tag_based_naming_access/param.i
+         block=Postprocessors
+         id=controls_tags
+         caption=Example of the parameter control_tags.
+
+The two postprocessors in [controls_tags] declare the same control tag `tag`.
+Thus their controllable parameter `point` can be set by controls simultaneously with `tag/*/point` as in [controls_tags_use].
+
+!listing test/tests/controls/tag_based_naming_access/param.i
+         block=Postprocessors
```

Comment:
```suggestion
         block=Controls
```

Reply:
@lindsayad if we make friend more specific as suggested, I am getting a compiling error
```
error: use of undeclared identifier 'ActionWarehouse'; did you mean 'OutputWarehouse'?
```
Unfortunately we cannot include `ActionWarehouse.h` in `InputParameters.h` because of the cyclic including I think. Any suggestions?